data/releases.yml

```diff
@@ -321,6 +321,26 @@
             - name: Install via npm
               link: https://npmjs.com/package/skywalking-client-js
 
+    - name: SkyWalking Rust
+      icon: kong-agent
```

Comment:
@Jtrust I am using `kong-agent` icon for now, but need a `rust-agent` icon. Please add one when you have time.